libslide/src/bignum/bignum_add.rs

```diff
@@ -0,0 +1,143 @@
+mod bignum_add {
+    use crate::bignum::Bignum;
+    use std::ops;
+
+    // @todo - figure out a swap method to handle lines 17-26
+    impl ops::Add for Bignum {
+        type Output = Bignum;
+
+        fn add(self, rhs: Bignum) -> Bignum {
+            // this assumes the signs are both positive for now
```

Comment:
If they are negative, just do (-1 * (-self + -rhs)). If one is negative and one is positive use subtraction.

Reply:
yep thats the plan